index.js

```diff
@@ -22,51 +19,79 @@ const disable = () => {
   };
 };
 
+/* eslint-disable no-unused-vars */
 const enable = () => {
   window.onscroll = () => {};
 };
 
-navToggle.addEventListener('click', () => {
+const popObjects = {
+  pop: {
+    title: document.querySelector('.hero1 h1').innerText,
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.",
+    image: document.querySelector('.img img').src,
+    tech: document.querySelector('#ul ul li').innerHTML,
+    live: 'https://OmarMwarraich.github.io/Portfolio',
+    source: 'https://github.com/OmarMWarraich/Portfolio',
+  },
+};
```

Comment:
[ OPTIONAL ] As I can see in your js file you created one object inside object but in the requirements they asked to create array of objects so you can display in each card differnent information so please fix that 🔥✅✅


![array](https://user-images.githubusercontent.com/58553711/181749699-2109b58e-6cda-4a39-bea0-14a555002fc6.PNG)

Reply:
@Reem-lab I will certainly make the necessary corrections.